packages/core/src/analytics/integrations/Omnitracking/omnitracking-helper.js

```diff
@@ -321,6 +321,25 @@ export const getValParameterForEvent = (valParameters = {}) => {
   return JSON.stringify(valParameters);
 };
 
+/**
+ * Get the product unit sale price from a specific product.
+ *
+ * @param {object} productData - Product data.
+ *
+ * @returns {*} The unit sale price value.
```

Comment:
This should be something like this since this function returns a number or undefined:
```suggestion
 * @returns {number|undefined} The unit sale price value, or undefined if no pricing data was found in productData argument.
```

Reply:
Fixed 👍🏼